### ISSUE TITLE
Cannot read properties of undefined (reading 'toLowerCase')

### ISSUE THREAD

**Issue by threeel** [VemtoOrg/vemto2-issues#38]
**Error Message:**
Cannot read properties of undefined (reading 'toLowerCase')

**Stack:**
```
TypeError: Cannot read properties of undefined (reading 'toLowerCase')
    at Function.plural (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:63:9899)
    at vl.pluralize (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:89:13558)
    at Proxy.tableNameIsDifferentFromDefault (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:91:8816)
    at Object.eval (eval at compileAsync (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:303:649), <anonymous>:94:16)
    at lae.compileAsync (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:303:675)
    at lae.compileAsyncWithErrorTreatment (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:303:290)
    at Oy.compile (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:349:14378)
    at Oy.compileWithImports (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:349:13975)
    at async SR.compile (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:351:3174)
    at async SR.compileWithErrorThreatment (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.38a36947.js:351:2295)
```

**Template Error Line:**
```
64
```

**Template Section:**
```

```

**How to reproduce:**
1. Create an empty model 
2. Try to delete
3.  call Save

**Expected behavior:**
Remove it.

**Actual behavior:**
Get the Error above

**Additional context:**

---

Vemto 2.0.5
darwin 23.3.0
Locale: en-GB
App version: 2.0.5

**Comment by TiagoSilvaPereira** [comment]
@threeel thanks for reporting

**Comment by TiagoSilvaPereira** [comment]
Fixed on Vemto 2.0.10
